packages/block-library/src/navigation-link/edit.js

```diff
@@ -147,12 +148,23 @@ function NavigationLinkEdit( {
 					/>
 				</PanelBody>
 			</InspectorControls>
-			<div className={ classnames(
-				'wp-block-navigation-link', {
-					'is-editing': isSelected || isParentOfSelectedBlock,
-					'is-selected': isSelected,
-					'has-link': !! url,
-				} ) }
+			<div
+				className={ classnames(
+					'wp-block-navigation-link', {
+						'is-editing': isSelected || isParentOfSelectedBlock,
+						'is-selected': isSelected,
+						'has-link': !! url,
+						'has-child': hasDescendants,
+						'has-text-color': navigationBlockAttributes.valueTextColor,
```

Comment:
Can `navigationBlockAttributes` be destructured to make this a little easier to grok? Or are these variable names already in use?

Reply:
Agree. let me rename or destructure it